### ISSUE TITLE
PostGis as database_vendor

### ISSUE THREAD

**Issue by tned73** [3YOURMIND/django-migration-linter#169]
Dear 3YOURMIND,

thanks again for the great work, I am working with Django in combination with a PostGis db. Therefor the next line is in our config
```
DATABASES = {
    "default": {
        "ENGINE": "django.contrib.gis.db.backends.postgis",
...
```
As I was testing if I would get an error when setting a models.Field with an extra argument `db_index=True`. This did not happen, and when debugging this issue was found that it would skip the `PostgresqlAnalyser` due to the fact that de vendor did not contain "postgre"
as mentioned in issue #138 The selection of an analyser is ambiguous. I would rather have it fail on me for an unknown db backend, then silently trying the best it can. And if it is an unknown backend then have a command_line/config argument to force a specific analyser so there is an option to select. 
Maybe it should not even try to detect, let it be command_line/config argument.

The point I want to make is, I want to trust the errors and warnings from this great package and that they are preventing me having deploy issues. But if it is internally attempting to select the correct analyser, it may select the wrong one but still give the impression that everything is fine. But if there is a backwards compatibility migration issue, not detected due to the for mentioned issue, I now have a deploy issue on my hands.

**Comment by David-Wobrock** [comment]
Hi @tned73 

Thanks for opening an issue.

I think it's time to tackle this, and I like a lot the idea of failing, but allowing selecting the SQL analyse oneself.
I opened https://github.com/3YOURMIND/django-migration-linter/pull/195
